### PR TITLE
Api 1.3 cleanup per api guidelines

### PR DESCRIPTION
**Pull request by mitchell852**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1336/
Test PASSed.

File: traffic_ops/traffic_ops_golang/routes.go [hunk 1]
```diff
@@ -71,55 +71,35 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		//About
 		{1.3, http.MethodGet, `about/?(\.json)?$`, about.Handler(), auth.PrivLevelReadOnly, Authenticated, nil},
 
-		// Proxied routes
```
Comment (mitchell852) [inline traffic_ops/traffic_ops_golang/routes.go]: no need for these anymore because the http.MethodGet, `cdns/{id}$`, route was removed

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1338/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1339/
Test PASSed.

**Pull request closed without merging**